media/css/firefox/pocket.scss

```diff
@@ -38,6 +37,9 @@ $image-path: '/media/protocol/img';
     margin-bottom: 0;
     margin-top: $spacing-xl;
 
+    // override the default rule on split
+    overflow-x: visible;
```

Comment:
Added these changes, and it's a good fix, but could probably be improved since the image is much larger than in prod. Not sure if it's blocking though. Let me know what you think!